src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/events/EventPublisher.java

```diff
@@ -4,5 +4,8 @@
 
 public interface EventPublisher {
 
+    String BULK_SCANNED = "Bulk_Scanned";
+    String EXCEPTION_RECORD = "ExceptionRecord";
```

Comment:
Are those names of events?
Would be nice to either rename or put in a class that indicates what those strings are IMHO.

Reply:
🛑 
Scrolling down in PR, turns out those are case type IDs apparently.
Indicate somehow what those are for sure.

Reply:
I'll rename as I need to put a comment they are temporary anyway. Need ccd client to be updated where we can query those case types. @timja explained/reminded it's temporary hack

Reply:
just moving them to `CaseTypeIds` seems cleanest to me